src/panels/lovelace/hui-view.ts

```diff
@@ -6,6 +6,7 @@ import {
   TemplateResult,
 } from "lit-element";
 
+import "@polymer/paper-fab/paper-fab";
```

Comment:
not needed?

Reply:
Why not?

Reply:
This was the only change to this file, no paper fab added

Reply:
I noticed it was used in that file, but not imported